source/_docs/locked_out.md

```diff
@@ -15,19 +15,60 @@ or need to recover your data.
 
 ## Forgot username
 
-If you’ve forgotten your username, ask the owner to help you.
-If you are using the {% term "Home Assistant Operating System" %} and have access to the Home Assistant server, you can connect a terminal and enter the `auth list` command. This command lists all users that are registered on your Home Assistant.
+### Symptom: I'm not the owner and I forgot my username
+
+You are **not the owner** of the Home Assistant server and you cannot login to Home Assistant because you forgot your username.
+
+#### Remedy
+
+Ask the owner to help you.
+
+### Symptom: I'm the owner and I forgot my username
+
+You are the **owner** of the Home Assistant server and you cannot login because you forgot your username.
+
+#### Remedy
+
+1. Check if the following conditions are met:
+   - you are using the {% term "Home Assistant Operating System" %}
+   - you have access to the Home Assistant server.
+2. Open a terminal connection to Home Assistant:
+   - If you are using a Home Assistant Green, follow these steps [to access the console](https://green.home-assistant.io/guides/use-terminal/).
+   - If you are using a Home Assistant Yellow, follow these steps:
+     - [to access the console from Windows](https://yellow.home-assistant.io/guides/use-serial-console-windows/)
+     - [to access the console from Linux or macOS](https://yellow.home-assistant.io/guides/use-serial-console-linux-macos/).
+   - If you are using another system, connect keyboard and monitor. The procedure might be similar the one used for Green.
```

Comment:
Maybe we should also mention virtualization here, since almost a third of our users use that:

If you are using a Home Assistant OVA (virtualization image), access the system console by opening the terminal through your virtualization platform's interface (e.g., VMware, VirtualBox). Follow the platform-specific steps to interact with the virtual machine's console.

or something like that.